optimum/quanto/models/diffusers_models.py

```diff
@@ -177,8 +176,8 @@ def from_pretrained(cls, model_name_or_path: Union[str, os.PathLike], **kwargs):
         model.eval()
         return cls(model)
 
-    def save_pretrained(self, save_directory: Union[str, os.PathLike], max_shard_size: Union[int, str] = "10GB"):
-        self._wrapped.save_pretrained(save_directory, max_shard_size=max_shard_size)
+    def _save_pretrained(self, save_directory: Path) -> None:
```

Comment:
Is this called internally by `ModelMixin`? 

You have already mentioned that we're losing access to `max_shard_size` but why does it have to be that?

Reply:
Yes it is. This guide explains you how that works: https://huggingface.co/docs/huggingface_hub/guides/integrations#a-more-complex-approach-class-inheritance. Basically by using the mixin you benefit from a few built-in stuff but on the other hand you have to comply to some constraints. You cannot add arguments here for instance. I'd say it's better like this to have more uniformization in signatures in the ecosystem.

Reply:
Not a strong opinion but libraries may have constraints beyond enforcements. But I understand how that kind of uniformization could be beneficial.